packages/babel-traverse/src/scope/lib/renamer.js

```diff
@@ -105,7 +105,7 @@ export default class Renamer {
     let { binding, oldName, newName } = this;
     let { scope, path } = binding;
 
-    let parentDeclar = path.find((path) => path.isDeclaration() || path.isFunctionExpression());
+    let parentDeclar = path.find((path) => path.isDeclaration());
```

Comment:
If we are removing this, do we still need ` this.maybeConvertFromClassFunctionExpression(parentDeclar);` below?